demo/grid-columns-demos.html

```diff
@@ -114,26 +114,23 @@ <h3>Hiding Columns</h3>
     </p>
     <vaadin-demo-snippet id='grid-columns-demos-hiding-columns'>
       <template preserve-content>
-        <vaadin-checkbox>Hide First Two Columns</vaadin-checkbox>
-
+        <vaadin-context-menu>
+          <vaadin-button aria-label="Hide column menu">
+            <iron-icon icon="lumo:menu"></iron-icon>
+          </vaadin-button>
+        </vaadin-context-menu>
         <vaadin-grid aria-label="Hiding Columns Example">
-          <vaadin-grid-column width="60px" flex-grow="0" text-align="end" header="#"></vaadin-grid-column>
-          <vaadin-grid-column width="60px" flex-grow="0" text-align="center"></vaadin-grid-column>
-          <vaadin-grid-column path="name.first" width="20%" flex-grow="0"></vaadin-grid-column>
-          <vaadin-grid-column path="name.last" width="20%" flex-grow="0"></vaadin-grid-column>
-          <vaadin-grid-column path="location.city" width="33%"></vaadin-grid-column>
-          <vaadin-grid-column path="location.state" width="33%"></vaadin-grid-column>
-          <vaadin-grid-column path="location.street" width="33%"></vaadin-grid-column>
+          <vaadin-grid-column width="60px" flex-grow="0" text-align="end" header="#" name="Index"></vaadin-grid-column>
+          <vaadin-grid-column width="60px" flex-grow="0" text-align="center" name="Picture"></vaadin-grid-column>
+          <vaadin-grid-column path="name.first" width="20%" flex-grow="0" name="First Name"></vaadin-grid-column>
+          <vaadin-grid-column path="name.last" width="20%" flex-grow="0" name="Last Name"></vaadin-grid-column>
+          <vaadin-grid-column path="location.city" width="33%" name="City"></vaadin-grid-column>
+          <vaadin-grid-column path="location.state" width="33%" name="State"></vaadin-grid-column>
```

Comment:
One column (maybe "State") could be pre-hidden (with `hidden`) in the demo.

Reply:
Done